examples/custom-engine-types/src/main.rs

```diff
@@ -408,7 +408,7 @@ where
         // but any custom logic can be implemented here
         reth_ethereum_payload_builder::EthereumPayloadBuilder::new(
             EthEvmConfig::new(chain_spec.clone()),
-            EthereumBuilderConfig::new(default_extra_data_bytes()),
+            EthereumBuilderConfig::new(default_extra_data_bytes(), ETHEREUM_BLOCK_GAS_LIMIT),
```

Comment:
this can be avoided by making this the default, remove from new and add a `with_gas_limit` builder style fn